src/navigationStore.js

```diff
@@ -195,15 +197,17 @@ function createNavigationOptions(params) {
     };
     const res = {
       ...props,
-      headerTintColor: navBarButtonColor || props.tintColor || navigationParams.tintColor || navigationParams.headerTintColor,
-      headerTitleStyle: headerTitleStyle || titleStyle,
-      title: getValue(navigationParams.title || title || getTitle, state),
+      headerBackImage: navigationParams.backButtonImage || backButtonImage,
       headerBackTitle: getValue(navigationParams.backTitle || backTitle, state),
-      headerRight: getValue(navigationParams.right || right || rightButton || params.renderRightButton, state),
+      headerBackTitleEnabled: navigationParams.backTitleEnabled || backTitleEnabled,
+      headerLayoutPreset: navigationParams.headerLayoutPreset || headerLayoutPreset,
       headerLeft: getValue(navigationParams.left || left || leftButton || params.renderLeftButton, state),
-      headerTitle: getValue(navigationParams.renderTitle || renderTitle || params.renderTitle, state),
```

Comment:
Any reason to remove headerTitle?

Reply:
It was not removed - just ordered those items alphabetically.